manifests/manifests.go

```diff
@@ -0,0 +1,379 @@
+package manifests
+
+import (
+	"context"
+	"encoding/json"
+	"io"
+
+	"github.com/containers/buildah/pkg/manifests"
+	"github.com/containers/buildah/pkg/supplemented"
+	cp "github.com/containers/image/v4/copy"
+	"github.com/containers/image/v4/docker/reference"
+	"github.com/containers/image/v4/image"
+	"github.com/containers/image/v4/manifest"
+	"github.com/containers/image/v4/signature"
+	is "github.com/containers/image/v4/storage"
+	"github.com/containers/image/v4/transports"
+	"github.com/containers/image/v4/transports/alltransports"
+	"github.com/containers/image/v4/types"
+	"github.com/containers/storage"
+
```

Comment:
Remove blank line

Reply:
Yeah, that shouldn't have been in there.